docs/csharp/programming-guide/classes-and-structs/structs.md

```diff
@@ -7,44 +7,36 @@ helpviewer_keywords:
 ms.assetid: b7cf4ff2-0eb7-4e5c-93d5-b2196b4f5d89
 ---
 # Structs (C# Programming Guide)
-Structs are defined by using the [struct](../../../csharp/language-reference/keywords/struct.md) keyword, for example:  
+
+Structs are defined by using the [struct](../../language-reference/keywords/struct.md) keyword, for example:  
   
- [!code-csharp[csProgGuideObjects#39](../../../csharp/programming-guide/classes-and-structs/codesnippet/CSharp/structs_1.cs)]  
+[!code-csharp[csProgGuideObjects#39](./codesnippet/CSharp/structs_1.cs)]  
   
- Structs share most of the same syntax as classes, although structs are more limited than classes:  
+Structs share most of the same syntax as classes, although structs are more limited than classes:  
   
--   Within a struct declaration, fields cannot be initialized unless they are declared as const or static.  
-  
--   A struct cannot declare a default constructor (a constructor without parameters) or a finalizer.  
-  
--   Structs are copied on assignment. When a struct is assigned to a new variable, all the data is copied, and any modification to the new copy does not change the data for the original copy. This is important to remember when working with collections of value types such as Dictionary\<string, myStruct>.  
-  
--   Structs are value types and classes are reference types.  
-  
--   Unlike classes, structs can be instantiated without using a `new` operator.  
-  
--   Structs can declare constructors that have parameters.  
-  
--   A struct cannot inherit from another struct or class, and it cannot be the base of a class. All structs inherit directly from `System.ValueType`, which inherits from `System.Object`.  
-  
--   A struct can implement interfaces.  
-  
--   A struct can be used as a nullable type and can be assigned a null value.  
+- Within a struct declaration, fields cannot be initialized unless they are declared as const or static.  
+- A struct cannot declare a default constructor (a constructor without parameters) or a finalizer.  
+- Structs are copied on assignment. When a struct is assigned to a new variable, all the data is copied, and any modification to the new copy does not change the data for the original copy. This is important to remember when working with collections of value types such as `Dictionary<string, myStruct>`.  
+- Structs are value types and classes are reference types.  
```

Comment:
nit unrelated to your PR: this should be something like, "Structs are value types, unlike classes, which are reference types."